src/sentry/auth/superuser.py

```diff
@@ -68,7 +68,9 @@
 
 UNSET = object()
 
-ENABLE_SU_UPON_LOGIN_FOR_LOCAL_DEV = getattr(settings, "ENABLE_SU_UPON_LOGIN_FOR_LOCAL_DEV", False)
+DISABLE_SU_FORM_U2F_CHECK_FOR_LOCAL = getattr(
```

Comment:
In general I'd recommend not using negative words like `disable` when naming bools. it can make for interesting to read logic like `if not DISABLE_SU_FORM_U2F_CHECK_FOR_LOCAL:` where if it's the positive intent, `if not ENABLE_SU_FORM_U2F_CHECK_FOR_LOCAL:` reads a bit more easily (and makes sense as `DISABLED` logically).

Could we just name it `ENABLE_SU_FORM_U2F_CHECK_FOR_LOCAL` and set it to `True`? (the other line would need to update to `note ENABLE_SU_FORM_U2F_CHECK_FOR_LOCAL` with that to maintain the logic too)

Reply:
That setting has actually existed for a while now and was created a couple years ago. It's a bit tedious to swap now b/c we're using it in BE and FE 😅